Doc/library/argparse.rst

```diff
@@ -1525,6 +1525,34 @@ be achieved by specifying the ``namespace=`` keyword argument::
 Other utilities
 ---------------
 
+Manpage
+^^^^^^^
+
+``argparse`` now have the ability to auto generate Unix man page.
+
+.. versionadded:: 3.7
+
+
+.. method:: ArgumentParser.add_manpage_section(heading, description)
+
+    Adding new section to man page.
+
+    Description of parameters:
+
+      * heading: title of the section, it will automatically convert to upper case
+      * description: description about this section
+
+Some example usage::
+
+>>> # Adding Copyright section
+>>> parser = argparse.ArgumentParser(prog='PROG')
+>>> parser.add_manpage_section('Copyright', 'Copyright \(co 2017 Python Software Foundation.')
```

Comment:
You are right, using `r'Copyright'` is more generic, but the backslash is still need, since this is using TROFF format, it will convert to `(c)` sign.